src/main/java/org/medicmobile/webapp/mobile/MedicAndroidJavascript.java

```diff
@@ -291,7 +296,11 @@ public String getDeviceInfo() {
 					.put("ram", ramObject)
 					.put("network", networkObject)
 					.toString();
-		} catch(Exception ex) {
+		} catch (RuntimeException e) {
```

Comment:
I think we actually might want to just suppress the warning here.  From the docs:

>It is a common bug pattern to say try { ... } catch (Exception e) { something } as a shorthand for catching a number of types of exception each of whose catch blocks is identical, but this construct also accidentally catches RuntimeException as well, masking potential bugs.

But, this is not what this code is actually trying to do.  If I understand this correctly we never want to throw any exceptions (including `RuntimeException`s) out of this method, but instead should always be logging the exception and returning the `jsonError` (so that downstream code can handle the error).

Reply:
👍Done